marklogic-data-hub-central/ui/src/components/entities/entity-tiles.test.tsx

```diff
@@ -142,13 +142,15 @@ describe("Entity Tiles component", () => {
       await fireEvent.click(customerPanel);
       // Check for Mapping tab
       expect(getByText('Map')).toBeInTheDocument();
-      expect(getByText('Custom')).toBeInTheDocument();
+      await fireEvent.click(getByText('Custom'));
+      expect(getByText("You can create Custom steps either manually or using Gradle, then deploy them. Deployed Custom steps appear here. Hub Central only allows running Custom steps, not editing or deleting them.")).toBeInTheDocument();
       // Check for Matching tab
       expect(queryAllByText('Match')).toHaveLength(0);
       await fireEvent.click(customerPanel);
 
       await fireEvent.click(NoEntityTypePanel);
       //Custom card 'customXML' should be selected by default
       expect(getByText('customXML')).toBeInTheDocument();
+      expect(getByText('Steps that are created outside Hub Central and are not associated with any entity type appear here. Hub Central only allows running these steps, not editing or deleting them.')).toBeInTheDocument()
```

Comment:
Similar to the above but:

aria-label={'customNoEntityTitle'}

and:

expect(getByLabelText('customNoEntityTitle')).toBeInTheDocument();

Reply:
Updated!!